src/Collapse.js

```diff
@@ -164,10 +166,14 @@ export class Collapse extends React.Component {
 
 
   render() {
-    const {theme, children} = this.props;
+    const {accessibilityId, theme, children, isOpened} = this.props;
     return (
-      <div ref={this.onRefContainer} className={theme.collapse} style={this.initialStyle}>
-        <div ref={this.onRefContent} className={theme.content}>
+      <div
+        ref={this.onRefContainer}
+        className={theme.collapse}
+        style={this.initialStyle}
+        aria-hidden={!isOpened}>
+        <div ref={this.onRefContent} className={theme.content} id={accessibilityId}>
```

Comment:
Why id on content element? Why not set it in your children’s parent element? What its purpose in a11y context? I can see how area-hidden can be useful, but very much against adding any extra special props, especially when they can be avoided

Reply:
The value one passes to `id` needs to match the controlling element's `aria-controls` value. We can leave it to the consumer to apply it as part of the child's top-level element, but I thought it'd be cool to encourage accessibility standards by exposing the optional prop.

Lemme know if you're still against the idea of it as a prop. I can edit the examples and documentation accordingly.

Reply:
@nkbt curious on your thoughts. You may be adding a prop to your library, but the end result for the consume is the same if a user follows best practices with accessibility. This way, you're simply encouraging best practices and helping to create a more accessible web.

Reply:
Look, I am very much against adding any props to the library unless it is absolutely impossible to do outside of the library and it is breaking intended functionality. I can understand the need for `aria-hidden` especially as it does not affect public API of the component itself at all, so it should be ok to add (though technically it still would be possible to overcome from outside, but the change is too unobtrusive). Everything else I don't see as part of the library, unfortunately, and would really like to avoid adding

Thank you for the change and all the reasoning though, I do appreciate your time and dedication in making web better. Legend 👍

Reply:
Sad to hear, but I've made adjustments to something I think you'll be happy with.